src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,39 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import './todoInfo.scss';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../Types/Todo';
+
+type Todos = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Todos> = ({ todo }) => {
+  const {
+    title,
+    user,
+    completed,
+  } = todo;
```

Comment:
Why don't you have id, too? You are using it below

Reply:
Done!